packages/orchestrator-ui-components/src/components/WfoTree/treeUtils.ts

```diff
@@ -21,9 +21,18 @@ export function getWfoTreeNodeDepth(
 }
 
 export const sortTreeBlockByLabel = (
-    { label: labelA }: TreeBlock,
-    { label: labelB }: TreeBlock,
+    { label: labelA, id: idA }: TreeBlock,
+    { label: labelB, id: idB }: TreeBlock,
 ): number => {
+    if (labelA === labelB) {
+        if (idA < idB) {
+            return -1;
```

Comment:
I assume that only < and > are defined and you cant just do a 'return idA - idB' ?

Reply:
I'm not sure what you mean by only < and > are defined? The sort function only needs to determine if numbers are  greater than, smaller than or equal to each other. Returning 1, -1, or 0 is just a convention I think is clearer